plugin/evm/message/codec.go

```diff
@@ -22,7 +24,7 @@ var (
 
 func init() {
 	Codec = codec.NewManager(maxMessageSize)
-	c := linearcodec.NewCustomMaxLength(maxMessageSize)
+	c := linearcodec.NewDefault(time.Time{})
```

Comment:
I guess these should be created with actual durango times? (since they were using custom max lenght before?)

Reply:
`maxMessageSize` is still enforced by the codec manager. So we would need to be serializing 0-length types for this to change the validity rules